admin-manual/installation/linux/ubuntu-xenial.rst

```diff
@@ -515,7 +527,7 @@ password ``12345`` and the permissions needed for the database created above.
 
 .. code-block:: bash
 
-   mysql -h localhost -u root -p -e "GRANT ALL ON atom.* TO 'atom'@'localhost' IDENTIFIED BY '12345';"
+   sudo mysql -h localhost -u root -p -e "GRANT ALL ON atom.* TO 'atom'@'localhost' IDENTIFIED BY '12345';"
 
```

Comment:
I think we are missing the section on SQL modes that we added to the Bionic / 18.04 docs - we should add it here too.